ts/store/reducers/deepLink.ts

```diff
@@ -26,13 +28,14 @@ export default (
   action: Action
 ): DeepLinkState => {
   switch (action.type) {
-    case SET_DEEPLINK:
+    case getType(setDeepLink):
       return {
         ...state,
-        deepLink: action.payload
+        deepLink: action.payload.navigationPayload,
+        immediate: action.payload.immediate
```

Comment:
@DenisFrezzato this is a bugfix?

Reply:
Actually, I don't know :smile: 
Refactoring I noticed that the action creator `setDeepLink` was accepting this `immediate` parameter, but it wasn't handled by the reducer. Apparently it's useless, so the options was:

- just set the `immediate` parameter to the state;
- remove all the `immediate` related code.

I chose the first option in order to stick to this PR.